pom.xml

```diff
@@ -30,7 +30,7 @@
 
   <groupId>org.apache.velocity.tools</groupId>
   <artifactId>velocity-tools-parent</artifactId>
-  <version>3.2-SNAPSHOT</version>
+  <version>4.0-jakarta-m001-SNAPSHOT</version>
```

Comment:
This is both the wrong branch and the wrong convention. We need to keep using the 2.0.x branch and the format should be `2.0.x-atlassian-jakarta-m001-SNAPSHOT`